src/mixins/filing-template-mixin.ts

```diff
@@ -29,18 +29,28 @@ export default class FilingTemplateMixin extends Vue {
   @Action setFilingId!: ActionBindingIF
   @Action setCertifyState!: ActionBindingIF
   @Action setShareClasses!: ActionBindingIF
+  @Action setFutureEffectiveDate!: ActionBindingIF
+  @Action setIsImmediate!: ActionBindingIF
+  @Action setIsFutureEffective!: ActionBindingIF
 
   /**
    * Method to construct a filing body when making an api request
    */
   buildFiling (): IncorporationFilingIF {
+    // Format DateTime for Filing
+    const effectiveDate = this.stateModel.incorporationDateTime.futureEffectiveDate
+    const formattedDateTime = effectiveDate &&
+      (effectiveDate.toISOString()).replace('Z', '+00:00')
```

Comment:
Need to verify that saved and restored date/time is OK with respect to local time. For example, if I specify "effective at 12:00PM (noon) today", does it get restored the same way? And is it correct in the Lear db? I ask this because I don't see conversions to/from localtime.

Reply:
The date is restored in the `deconstructDateTime` method where it converts it to local time. It is always saved into store and filed as UTC. This does line up with the backend ( afaik ) and the display of the time after being parsed from a draft doesn't change its actual value in store. But this is definitely worth more research, i could certainly be missing something.

Reply:
Can confirm it is restored in the same manner. Essentially local time is just used for the presentation, whereas everything being stored is UTC.